packages/camera/camera_web/example/build.yaml

```diff
@@ -0,0 +1,6 @@
+targets:
```

Comment:
Is this required for mocktail? I thought this was only required to generate mockito mocks?

Reply:
yeah it shouldn't be required for mocktail

Reply:
Removed 👌